db/migrations/20211004183106_clucks.js

```diff
@@ -0,0 +1,19 @@
+
+exports.up = function(knex) {
+    return knex.schema.createTable("clucks", table => {
+      
+      table.string('title');
+      table.text("imageUrl");
```

Comment:
fyi. there's a char constraint for text so if `imageUrl` is >256 characters db update won't succeed